.claude/skills/setup/SKILL.md

```
@@ -0,0 +1,106 @@
+```bash
+docker info > /dev/null 2>&1 && echo "Docker is running" || echo "Docker is not running"
+```
+
+### 2. Start All Services
+
+```bash
+docker compose up -d
+```
+
+This starts:
+- **Frontend**: Vue 3 app on http://localhost:8080
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)